Sources/PointFree/Routers/AccountRoutes.swift

```diff
@@ -0,0 +1,151 @@
+import ApplicativeRouter
+import ApplicativeRouterHttpPipelineSupport
+import Either
+import Foundation
+import HttpPipeline
+import Optics
+import Prelude
+import Styleguide
+import Tuple
+
+public let accountRouter = accountRouters.reduce(.empty, <|>)
```

Comment:
trying something new here. the account routes + middlewares + views are getting so big that I decided to split out a sub router for just account routes. figure we could do this with a few of our larger subpages. admin might be a good one to do next.